src/ain.ts

```diff
@@ -108,72 +125,65 @@ export default class Ain {
   }
 
   /**
-   * Returns the transaction with the given transaction hash.
-   * @param {string} transactionHash
+   * Fetches the information of a transaction with a transaction hash.
+   * @param {string} transactionHash The transaction hash.
    * @return {Promise<TransactionInfo>}
    */
   getTransaction(transactionHash: string): Promise<TransactionInfo> {
     return this.provider.send('ain_getTransactionByHash', { hash: transactionHash });
   }
 
   /**
-   * Returns the state usage information with the given app name.
-   * @param {string} appName
+   * Fetches the state usage information of a blockchain app.
+   * @param {string} appName The blockchain app name.
    * @return {Promise<StateUsageInfo>}
    */
   getStateUsage(appName: string): Promise<StateUsageInfo> {
     return this.provider.send('ain_getStateUsage', { app_name: appName });
   }
 
   /**
-   * Returns the result of the transaction with the given transaaction hash.
-   * @param {string} transactionHash
-   * @return {Promise<Transaction>}
-   */
-  // TODO(liayoo): implement this function.
-  // getTransactionResult(transactionHash: string): Promise<TransactionResult> {}
-
-  /**
-   * Validate the given app name.
-   * @param {string} appName
+   * Validates a blockchain app name.
+   * @param {string} appName The blockchain app name.
    * @return {Promise<AppNameValidationInfo>}
    */
   validateAppName(appName: string): Promise<AppNameValidationInfo> {
     return this.provider.send('ain_validateAppName', { app_name: appName });
   }
 
   /**
-   * Signs and sends a transaction to the network
-   * @param {TransactionInput} transactionObject
-   * @param {boolean} isDryrun - dryrun option.
+   * Signs and sends a transaction to the network.
+   * @param {TransactionInput} transactionObject The transaction input object. 
+   * @param {boolean} isDryrun The dryrun option.
    * @return {Promise<any>}
    */
   async sendTransaction(transactionObject: TransactionInput, isDryrun: boolean = false): Promise<any> {
     return this.signer.sendTransaction(transactionObject, isDryrun);
   }
 
   /**
-   * Sends a signed transaction to the network
-   * @param {string} signature
-   * @param {TransactionBody} txBody
-   * @param {boolean} isDryrun - dryrun option.
+   * Sends a signed transaction to the network.
+   * @param {string} signature The signature of the transaction.
+   * @param {TransactionBody} txBody The transaction body.
+   * @param {boolean} isDryrun The dryrun option.
    * @return {Promise<any>}
    */
   async sendSignedTransaction(signature: string, txBody: TransactionBody, isDryrun: boolean = false): Promise<any> {
     return this.signer.sendSignedTransaction(signature, txBody, isDryrun);
   }
 
   /**
-   * Sends signed transactions to the network.
-   * @param {TransactionInput[]} transactionObjects
+   * Sends signed multiple transactions in a batch to the network.
```

Comment:
How about "Signs and Sends multiple transactions ..." like `sendTransaction`? I think that user does not input signed transaction.

Reply:
Done, thanks!